inference/models/owlv2/owlv2.py

```diff
@@ -71,6 +82,26 @@ def _check_size_limit(self):
                 self.popitem(last=False)
 
 
+class Owlv2Singleton:
```

Comment:
I wonder if this can result in RAM exhaustion on long running processes that request multiple distinct models.